doc/getting-started.md

```diff
@@ -0,0 +1,344 @@
+## Getting started
+
+Your [Dogecoin Core installation](/INSTALL.md) is done, you now have `dogecoind` or `dogecoin-qt` executables available to run a node, and `dogecoin-cli`/`dogecoin-tx` tools but don't know how it works ?
+
+This tutorial will help you to go through the basics to use them !
+
+> **Warning :** For simplicity, explained commands assume executables are under `PATH` environment variable.  
+> If needed, specify their location by doing `PATH=$PATH:/path/to/executables` or use their path as command like :
+> ```bash
+> $ /path/to/dogecoin-cli [arguments ...]
+> ```
+
+### Table of contents
+
+1. [Run a dogecoin node](#run-a-dogecoin-node)
+2. [Introduction to JSON-RPC API](#introduction-to-json-rpc-api)
+    * [Create a wallet](#create-a-wallet)
+    * [Send a transaction](#send-a-transaction)
+    * [Inspect blocks and transactions](#inspect-blocks-and-transactions)
+3. [Node configuration](#node-configuration)
+    * [Mainnet, testnet and regnet](#mainnet-testnet-and-regnet)
+    * [RPC credentials](#rpc-credentials)
+    * [Ports](#ports)
+    * [Memory](#memory)
+    * [Mode debug](#mode-debug)
+
+## Run a dogecoin node
+
+To launch your node, you can choose `dogecoind` to run an headless server or `dogecoin-qt` for a server with a GUI.
+
+In a terminal, run :
+```bash
+# Headless node
+$ dogecoind -printtoconsole
+
+# GUI
+$ dogecoin-qt
+```
+
+*Use `--help` to see all available options for each executables.*
+
+Your node should now be running, starting a *syncing process* by downloading the entire blockchain from other nodes. This operation will take hours, days to complete... but your now part of the dogecoin network !
+
+> **Warning :** The rest of the guide will assume the use of an headless node. The RPC server is not exposed with `dogecoin-qt` until you activate the `-server` option.
+
+## Introduction to JSON-RPC API
+
+Dogecoin Core expose a JSON-RPC interface. With it, you can request information about blocks and transactions, send transactions to the networks, manage your wallet and so on...
+
+To interact with the interface, it relies on HTTP requests. Using urls like `http://user:password@127.0.0.1:22555/` and sending commands through json format is feasible.
+
+Some tools like libraries can be built on top of it. `dogecoin-cli` is the default shell client provided with Dogecoin Core to use this API.
+
+The RPC API documentation is available from [bitcoin](https://developer.bitcoin.org/reference/rpc/).  
+To have an overview of all commands, see also :
+
+```bash
+# List all commands
+$ dogecoin-cli help
+
+# Get help for a specific command
+$ dogecoin-cli help COMMAND
+```
+> **Tips** : Abuse of the help menus is allowed ! They can answer to multiple questions.
+
+### Create a wallet
+
```

Comment:
May be interesting to add `importprivkey`

Reply:
1.14.4 added a GUI for that, just saying....

Reply:
? Doing this through the CLI is useful too

Reply:
But it's the command used to do this, the problem is more the way to use it. I totally agree with the security risk, but I suppose not showing it won't solve the problem.

In a scenario where someone is playing with the CLI and want to import some keys, he will google it and find this info. Explained or not, these risks exists. Don't you think it's more interesting to show the command and explain briefly associated risks ?

Also, devs may need this information without being exposed to real risk in a development environment.

Reply:
> Don't you think it's more interesting to show the command and explain briefly associated risks ?

It depends on the intended audience for the document. If I look at it from a getting started perspective, then I'd leave it out and instead just note that it's important to make wallet.dat backups. Perhaps it's an idea to implement sweeping of private keys rather than import, and then list that instead of importing them as a best practice? I.e. leave it out for now, build sweep into 1.14.6's RPC, document that, then build it into the UI under Operation Such Frenzy and we're Gucci?

Reply:
Agree to keep it has it, it's a good starting point and this information is not hard to find. But yeah, I was targeting more blockchain devs rather than basic Dogecoin users.